distributed/worker.py

```diff
@@ -200,7 +207,7 @@ def __init__(self, key, runspec=None):
         self.dependencies = set()
         self.dependents = set()
         self.duration = None
-        self.priority = None
+        self.priority: tuple[int, ...] | None = None
```

Comment:
I don't have a super strong opinion but I prefer this approach a bit more because declaration and initialization is closer together. Fewer places to change and look at. However, we've been doing the C++ style thing in other places so I'm open to change this (I won't add types to other attributes for now, trying to keep things small-ish